libr/core/cmd_write.c

```diff
@@ -2068,6 +2068,10 @@ static int cmd_write(void *data, const char *input) {
 	{
 		size_t len = core->blocksize;
 		const char *curcs = r_config_get (core->config, "cfg.charset");
+
+		r_str_trim_args ((char *)input);
```

Comment:
better strdup (input). dont assume the passed string is modificable because its received as const

Reply:
yes.